php-apache/usr/local/share/php/common_functions.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+do_build_permissions() {
+  if [ "$IS_NFS" -ne 0 ]; then
+    setfacl -R -m "d:u:$CODE_OWNER:rwX" -m "u:$CODE_OWNER:rwX" /app/
```

Comment:
sadly GCE doesn't support setfacl, as we just found out, so this will have to be chown